distribution/lib/Standard/Table/0.0.0-dev/src/Extensions/Table_Conversions.enso

```diff
@@ -83,6 +83,11 @@ XML_Element.to_table self =
      Otherwise, treated as a single row.
    - fields: a Vector of Text representing the names of fields to look up.
      If `Nothing` then all fields found are added.
+   - treat_dictionary_as_sequence: If set to `True`, then a dictionary (and
```

Comment:
If this applies to dictionaries and other types as well, perhaps `treat_as_sequence` would be more precise.

Reply:
Perhaps, but I don't think such name is good. Regardless of the setting, we always treat e.g. a `Table` or `Column` as a sequence. It would be misleading to have `treat_as_sequence=False` and then most types are still treated as sequence...

This setting deals only with the less-sequence-like types, mostly dictionary-like. Perhaps `try_treating_all_types_as_sequence` could be a more precise name, what do you think? I'm worried it will be a bit too scary for users though... This setting overall is a bit complex. Maybe we shouldn't expose it publicly after all?

Reply:
I see your point, `treat_dictionary_as_sequence` is good.

Reply:
The argument was removed from public API for now, will be reintroduced with new design as part of #11681